managed/src/main/java/com/yugabyte/yw/models/Universe.java

```diff
@@ -823,4 +823,15 @@ public void run(Universe universe) {}
     };
     Universe.saveDetails(universeUUID, updater);
   }
+
+  public static boolean existsCertificate(UUID certUUID, UUID customerUUID) {
+    Set<Universe> universeList = Customer.get(customerUUID).getUniverses();
+    for (Universe universe : universeList) {
```

Comment:
Nit: Maybe use the java stream functionality. (If unaware, take a look [here](https://github.com/yugabyte/yugabyte-db/blob/master/managed/src/main/java/com/yugabyte/yw/models/Schedule.java#L132))